site/content/integrate/apps/quickstart/quick-start-jwt/_index.md

```diff
@@ -2,17 +2,19 @@
 title: "JWT in Go"
 heading: "Use JWT in a Mattermost App in Go"
 description: "This quick start guide will walk you through the basics of using JWT in a Mattermost App in Go."
-weight: 10
+weight: 30
 ---
 
 This quick start guide will walk you through the basics of using [JSON Web Tokens (JWT)](https://jwt.io/) in a Mattermost App in Go. In this guide you will review an App that:
 
-- Contains a `manifest.json`, declares itself an HTTP application that acts as a bot, uses JWT, and attaches to locations in the Mattermost interface.
+- Contains a `manifest.json`, declares itself an HTTP application that uses a bot account, uses JWT, and attaches UI elements to locations in the Mattermost interface.
 - Attaches the `send` function to a `/hello-jwt` command and wraps it with a function to authenticate JWT.
 
+Why you would want to use JWT for App development? Using a token, your App can validate the legitimacy of any request verify that this request is indeed coming from the Mattermost server we are expecting it to come from. Simply put, implementing JWT will enhance your App's security and functionality.
```

Comment:
This doesn't read quite right. I think there's a comma missing before `verify`.

Reply:
I'll fix this one.